python/pylibcudf/pylibcudf/libcudf/interop.pxd

```diff
@@ -71,7 +71,7 @@ cdef extern from *:
     ArrowArray* to_arrow_host_raw(
       cudf::table_view const& tbl,
       rmm::cuda_stream_view stream        = cudf::get_default_stream(),
-      rmm::mr::device_memory_resource* mr = rmm::mr::get_current_device_resource()) {
+      rmm::device_async_resource_ref mr = rmm::mr::get_current_device_resource()) {
```

Comment:
```suggestion
      rmm::device_async_resource_ref mr = rmm::mr::get_current_device_resource_ref()) {
```

Or does that rely on the as-yet-unmerged RMM PR?

Reply:
Right, I fix this up in a later PR.  It actually becomes
```suggestion
      cudf::device_async_resource_ref mr = cudf:get_current_device_resource_ref()) {
```